cmd/bridge/main.go

```diff
@@ -503,6 +511,15 @@ func main() {
 		bridge.FlagFatalf("listen", "scheme must be one of: http, https")
 	}
 
+	srv.HelmChartRepoProxyConfig = &proxy.Config{
+		TLSClientConfig: &tls.Config{
+			InsecureSkipVerify: true,
```

Comment:
This can't be true. Specify a CA file

Reply:
@spadgett  would it be ok if `caFile` is optional? Should we have some default value? and would it be ok to allow disabling tls verification, just as we allow that when Prometheus is out of the cluster?

ATM, chart repository is external to cluster, but later we can run within the cluster as well.

Reply:
CA file can be optional, but we can't disable TLS verification

Reply:
> but we can't disable TLS verification

Fair enough, even with our new 'external chart repo' design, makes sense to not disable TLS verification.